server/models/request.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require("mongoose");
+
+// basic request model
+const requestSchema = new mongoose.Schema({
+  // References review owner
+  reviewOwner: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "Review",
+  },
```

Comment:
In this model, how do you know who the selected reviewer is?

Also, can we consider embedding the review in this request object, and then cleaning it up a bit. I'm thinking the request object needs:
- userOwner
- selectedReviewer
- requestCreateDate
- reviewersDeclined
- statuses (I don't know if you need past statuses though tbh, maybe just current status)
- embedded review object

review object just needs:
- title, language, list of messages (everything else is already in the request object)